packages/auth/jest.config.json

```diff
@@ -2,7 +2,7 @@
   "clearMocks": true,
   "transform": { "^.+\\.jsx?$": "../../jestTransformer.js" },
   "testMatch": ["<rootDir>/src/__tests__/**/**.test.js"],
-  "setupFilesAfterEnv": ["../../jestScaffolding.js","./jest.setup.js"],
+  "setupFilesAfterEnv": ["../../jestScaffolding.js","./src/__tests__/jestScaffolding.js"],
```

Comment:
I like the `jest.setup.js` file you originally used, we could even consider renaming `jestScaffolding`.js under the root to `jest.setup.base.js` or similar... So I suggest we keep it!